nimgame2/entity.nim

```diff
@@ -821,5 +821,71 @@ method onCollide*(entity, target: Entity) {.base.} =
   discard
 
 
+proc transform*(self:Entity):Transform=
```

Comment:
General formatting style notes, here and further:
* change `self` to `entity`, etc. to conform to the established style;
* don't forget spaces after `:` and before `=`;
* consider using templates instead of procs where it seems reasonable.

So, e.g. this line should be changed into `template transform*(entity: Entity): Transform =`, etc.

Reply:
I'll format the PR